### PR TITLE
fixing Request Body button in parameters.

### PR DESCRIPTION
**Pull request by Alexeyca**

### CODE REVIEW

File: src/components/api-request.js [hunk 1]
```diff
@@ -461,7 +461,7 @@ export default class ApiRequest extends LitElement {
                 <button class="tab-btn ${this.activeSchemaTab === 'body' ? 'active' : ''}" data-tab="body">BODY</button>
               </div>
               ${html`<div class="tab-content col" style="display: ${this.activeSchemaTab === 'model' ? 'block' : 'none'}"> ${reqBodySchemaHtml}</div>`}
-              ${html`<div class="tab-content col" style="display: ${this.activeSchemaTab === 'model' ? 'none' : 'block'}"> ${reqBodyExampleHtml}</div>`}
+              ${html`<div class="tab-content col" style="display: ${this.activeSchemaTab === 'body' ? 'none' : 'block'}"> ${reqBodyExampleHtml}</div>`}
```
Comment (wparad) [inline src/components/api-request.js:464]: This change doesn't seem right, what's the problem that this is fixing?

File: src/components/api-request.js [hunk 2]
```diff
@@ -539,9 +539,9 @@ export default class ApiRequest extends LitElement {
               : html`
                 ${fieldType === 'object'
                   ? html`
-                  <div class="tab-panel row" style="min-height:220px; border-left: 6px solid var(--light-border-color); align-items: stretch;">
+                  <div class="tab-panel row" style="min-height:300px; border-left: 6px solid var(--light-border-color); align-items: stretch;">
```
Comment (wparad) [inline src/components/api-request.js:542]: Can you share a bit about what's the reason for this change, maybe a couple of pictures or an example spec where we can trade off on what this size should actually be. Obviously it's hard coded for a reason so figuring out the right value for this is always a challenge.

**Review by wparad — COMMENTED** [review]
I have two questions about these changes, hopefully they make sense.

File: src/components/api-request.js (hunk 2 above)
Comment (Alexeyca) [reply src/components/api-request.js:542]: Did you try to use it? it not enough room for Request Body button, and that button move under schema

Comment (wparad) [reply src/components/api-request.js:542]: Do you have a sample spec I can use to validate the change?

Comment (Alexeyca) [reply src/components/api-request.js:542]: Screen Shot 2022-09-02 at 9 36 31 AM

Comment (Alexeyca) [reply src/components/api-request.js:542]: technically, it totally not usable

Comment (wparad) [reply src/components/api-request.js:542]: Next time I would appreciate a sample spec to validate this. From what I've been able to test it seems to okay.

**Review by wparad — APPROVED** [review]
I've approved this, but it won't be able to be merged until the CLA is signed.

**Comment by Alexeyca** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by Alexeyca** [comment]
I have read the CLA Document and I hereby sign the CLA
